public/contents/3-build-your-first-web-app/tours.json

```diff
@@ -35,7 +35,7 @@
       "fileSize": "32.13 KB",
       "fileName": "createuifromcollections-web.zip",
       "fid": 17161,
-      "checksum": "f4a220dcd25447979bc8baec7b9063bfe348cef8b3d738d743025631c11b9c3f",
```

Comment:
With fixes made in Hikes, the checksums have to be updated.

Reply:
I have updated the checksums for tour zips that I changed